### ISSUE TITLE
Make diagram on Mission and Structure a clickable image map

### ISSUE THREAD

**Issue by nutjob4life** [EDRN/P5#193]
Sudhir wants the image on https://edrn-dev.jpl.nasa.gov/portal/renaissance/about-edrn/mission-and-structure/ to be a clickable image map

**Comment by nutjob4life** [comment]
Ooof. This is going to be tough.

To make this happen, we'll need to:

1. Redraw the "org chart" as a SVG-format image and not the ghetto screenshot from Sudhir's presentation
2. Add `id` tags to the appropriate elements in the image that should be clickable
3. Install the `wagtail-svgmap` add-on
4. Add an ImageMap and, using the SVG chart, link the identified elements to page destinations in the portal
5. Edit the "Mission and Structure" page to replace the static image with the ImageMap

Image maps were supported by a [Wagtail add-on called "wagtail-svgmap"](https://github.com/City-of-Helsinki/wagtail-svgmap) but this add-on hasn't been supported in 4–6 years. It is not at all compatible with the versions of Wagtail (the CMS) and Django (the web framework) we're using.

Over the holiday thus far I've devoted myself to upgrading the add-on for compatibility with Wagtail 4 and Django 4. I was able to get _parts_ of it working, but ultimately hit a dead-end that depends on some deep-interactions between the add-on and Wagtail. I'm going to have to analyze a lot more or beg the original authors and/or the Wagtail authors for help on this.

For now, I've made the entire org chart a single clickable image.
